analysis/codelists.py

```diff
@@ -0,0 +1,36 @@
+from ehrql import codelist_from_csv
+
+acute_otitis_media_cod = codelist_from_csv(
+    "codelists/opensafely-pharmacy-first-acute-otitis-media-treatment-dmd.csv",
+    column="code",
+)
+
+first_impetigo_treatment_cod = codelist_from_csv(
+    "codelists/opensafely-pharmacy-first-impetigo-treatment-dmd.csv",
+    column="code",
+)
+
+infected_insect_bites_cod = codelist_from_csv(
```

Comment:
To avoid later confusion add word treatment here (as there is also a diagnosis code for same condition).
```suggestion
infected_insect_bites_treatment_cod = codelist_from_csv(
```

Reply:
thanks :) addressed in ca1e1573c933a290905a92e8d8cc6065fe151a77